src/main/kotlin/com/terraformation/backend/customer/model/TerrawareUser.kt

```diff
@@ -230,6 +230,8 @@ interface TerrawareUser : Principal {
 
   fun canReadInternalTags(): Boolean
 
+  fun canReadModulesForProject(projectId: ProjectId): Boolean
```

Comment:
This name feels a little inconsistent with the other permission names to me. What would you think of `canReadProjectModules`, to follow the same pattern as `canReadProjectVotes` and friends?